packages/order-watcher/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "2.2.2",
+        "changes": [
+            {
+                "note": "start jsonRpcRequestId at 1, not 0",
```

Comment:
Mind adding a bit more of "why"? Is there a PR/issue we could link to?

Reply:
```suggestion
                "note": "Start jsonRpcRequestId at 1, not 0 as 0 breaks Parity websocket RPC",
```